pkgs/games/grapejuice/default.nix

```diff
@@ -9,21 +10,21 @@
 , desktop-file-utils
 , xdg-utils
 , xdg-user-dirs
-, wine
+, gettext
 , winetricks
-, pciutils
-, glxinfo
+, wineWowPackages
+, ...
```

Comment:
```suggestion
```

unnecessary

Reply:
```
error: anonymous function at /home/helium/code/pr/nixpkgs/pkgs/games/grapejuice/default.nix:1:1 called with unexpected argument 'wine'

       at /home/helium/code/pr/nixpkgs/lib/customisation.nix:69:16:

           68|     let
           69|       result = f origArgs;
             |                ^
           70|
```

without that `nix-build -A grapejuice` fails with this 
(https://discourse.nixos.org/t/cant-upgrade-nixos-unexpected-argument-specialargs/13539/2)

Reply:
remove the wine override in all-packages.nix from grapejuice